.github/workflows/main.yaml

```diff
@@ -0,0 +1,30 @@
+# This is a basic workflow to build amd64/arm64 kubectl image for portainer
+
+name: build
+
+on:
+  push:
+    branches: develop
```

Comment:
the develop branch need to be pushed to portainerci repo instead.
we would also need to: 
- build feature branches as well, and push them as tags like: portainerci/kubectl-shell:pr-123
- build main branch and push to portainer/kubectl-shell:latest

Reply:
+ filter the folder to only trigger on kubectl-shell changes

Reply:
done